src/main/java/com/limechain/network/protocol/state/StateMessages.java

```diff
@@ -22,12 +23,12 @@ public StateMessages(String protocolId, StateProtocol protocol) {
     }
 
     public SyncMessage.StateResponse remoteStateRequest(Host us, AddressBook addrs, PeerId peer,
-                                                        String blockHash) {
+                                                        String blockHash, ByteString after) {
```

Comment:
Are there any occurrences of this being called with `AddressBook` that's not gotten from the `Host` object? If not then you can delete the second parameter.